src/screens/SiteDetailScreen.js

```diff
@@ -658,6 +780,44 @@ const styles = StyleSheet.create({
         flexDirection: 'row',
         marginTop: -15
     },
+    overlayContainer: {
+        flex: 1,
+    },
+    top: {
+        margin:20,
+        height: '30%',
+        alignItems: 'center',
+        justifyContent: 'center',
+        borderWidth:1,
+        borderColor: headerWhiteMediumTransparent,
+        paddingLeft: 10,
+        paddingRight: 10,
+        paddingTop:10,
+        paddingBottom:10,
+        backgroundColor: headerSemiWhiteTransparent,
+    },
+    facilities_features:{
+      backgroundColor:headerWhiteTransparent,
+      color:'white'
```

Comment:
Passing `color: 'white'` is throwing an error in the console. Color is an invalid prop for a `<ListItem />`. This line should be removed, and it looks like it works well.